### ISSUE TITLE
[FEATURE REQUEST] Ingame Editor

### ISSUE THREAD

**Issue by Coswics** [MehVahdJukaar/slotify#1]
I dont know how plausible this is, but having an ingame config with built in examples to show how stuff is moved around would be great, theres a way (idk how exactly) to include additional texture/resource packs on game load and the mod could just change those values instead of relying on user defined ones, and then reference the config generated instead of that?? maybe?? im jus spitballing but having some stuff set up would be good. (Inventory Profiles next has a good editor, so you could prolly ask the dev of that or reference how things go over there)

**Comment by MehVahdJukaar** [comment]
yeah of course would be nice but the amount of work needed to get this done would be huge compared to the mod itseld which started as just 2 mixins. The mod is meant for packmakers anyways and tweaking can already be done "ingame" by just altering the resourcepack slightly and reloading it with f3+h so thats the fastest way to achieve such results. the image in the screens was done in half an hour using this method
